.github/workflows/build.yml

```diff
@@ -0,0 +1,23 @@
+name: Build
+
+on: [ push, pull_request ]
```

Comment:
This might run the build twice, I think. 
We probably would need something more elaborate but this is a good start.

Reply:
Do you mean a case when someone creates a PR from this repo's branch instead of a fork? I guess you may be right. Perhaps something like this would do the trick:

```yaml
on:
  push:
    branches:
      - master
  pull_request:
```
🤔

Reply:
Yep, should be better 👍